Updated Quickstart Footers

codewithdev opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [contributing.md](/main/CONTRIBUTING.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

Closes #4849 

<!-- 
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed:

<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->
Updated the footers in Quickstart map topics,
 - [Set up Git](https://docs.github.com/en/github/getting-started-with-github/quickstart/set-up-git)
 - [Create a Repo](https://docs.github.com/en/github/getting-started-with-github/quickstart/create-a-repo)
 - [Fork a Repo](https://docs.github.com/en/github/getting-started-with-github/quickstart/fork-a-repo)
 - [Be Social](https://docs.github.com/en/github/getting-started-with-github/quickstart/be-social)

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

<!-- Description of the writer impact here -->

codewithdev committed:
Add Footer Create a Repo

codewithdev committed:
Added Footer Set up Git

codewithdev committed:
Added Footer Fork a Repo

codewithdev committed:
Update be-social.md

janiceilene commented:
🙌 You're on a roll @codewithdev 🎉 I'll get this triaged for review ⚡

felicitymay commented:
Hi @codewithdev - an internal project moved these articles to a new location, which caused these conflicts. I'll checkout your branch locally and see if I can resolve them.

felicitymay commented:
Hi @codewithdev - I haven't worked with forks for a while and had forgotten that it would be slightly more complicated to get the changes from `main`. Would you be able to update the `main` branch of your fork from the `docs` repository?

You can do this in the UI here: https://github.com/codewithdev/docs

![image](https://user-images.githubusercontent.com/1877141/122900829-e9e0e900-d344-11eb-91e1-f3aa11d4119b.png)

codewithdev commented:
Sure I would,

codewithdev commented:
@felicitymay Fortunately, my main branch is `even` with the upstream repository. Also, I can't update the branch for this PR as the path of the files has been changed and it has conflicts. Let me know what I can do? 🙄

felicitymay committed:
Merge branch 'main' into codewithdev-docs-2

felicitymay commented:
@codewithdev  - thanks for updating the `main` branch for your fork 🙇🏻‍♀️ 

I've pushed a commit that merges the changes from `main` into your branch as follows:

1. Made a local clone of your repository.
2. Updated my local copy of your `main` branch with the latest changes.
3. Checked out your branch.
4. Merged `main` into your branch.
5. I was expecting to need to resolve some conflicts, but Git was able to resolve the changes from `main` without help.
6. Pushed the resulting merge commit to your branch on GitHub.

I'll take a look at your changes soon.

felicitymay commented on content/get-started/quickstart/fork-a-repo.md:
```diff
@@ -148,7 +148,16 @@ Fork a repository to start contributing to a project. {% data reusables.reposito
+
+For more information see, "[Create a repository](/articles/create-a-repo)".
+
+Each repository has its own identity on {% data variables.product.prodname_dotcom %} that is associated with some people, or an organization. You can interact with people, repository on {% data variables.product.prodname_dotcom %}. for more information see, "[Be social](/articles/be-social)".
```
Updating to match the text in the article above ☝🏻 

```suggestion
Each repository in {% data variables.product.product_name %} is owned by a person or an organization. You can interact with the people, repositories, and organizations by connecting and following them on {% data variables.product.product_name %}. For more information see "[Be social](/articles/be-social)."
```

felicitymay commented on content/get-started/quickstart/fork-a-repo.md:
```diff
@@ -148,7 +148,16 @@ Fork a repository to start contributing to a project. {% data reusables.reposito
+
+Each repository has its own identity on {% data variables.product.prodname_dotcom %} that is associated with some people, or an organization. You can interact with people, repository on {% data variables.product.prodname_dotcom %}. for more information see, "[Be social](/articles/be-social)".
+
+You can also {% data reusables.support.connect-in-the-forum-bootcamp %} and ask your queries. 
```
This is difficult. You've used a "reusable" which is good for making articles consistent and easier to maintain. 😄 

However, this reusable was written as a bullet point - so it starts with a capital letter which makes it unsuitable for use within a sentence. 😞 

I've checked where this is used, and it is used only in these four articles that you're editing. So here I think the best solution would be:

1. Edit the text of the reusable so that it matches what you want to say in all of these articles. It is stored here: https://github.com/github/docs/blob/main/data/reusables/support/connect-in-the-forum-bootcamp.md. I'd recommend something like this, which is based on one of the articles above:
    `{% data variables.product.product_name %} has a great support community where you can ask for help and talk to people from around the world. Join the conversation on [Github Support Community](https://github.community/).`
2. Replace the current text about the support community in each of these articles with just `{% data reusables.support.connect-in-the-forum-bootcamp %}`.

codewithdev replied:
I actually missed that point. However, these reusable are quite handy in terms of adding into the context.

felicitymay reviewed (commented):
@codewithdev - thank you so much for taking the time to think about what a new user needs to know when they finish working through each of these articles. Your proposals will help them decide what to do next 💖 

I've added a lot of comments, but most of them are just for small changes. Since you're an active contributor, I've taken the time to add some explanations of most of the suggested changes. 

For smaller pull requests, we normally commit the changes immediately to speed up the turn around time for merging the work. Since this is a larger pull request and you've put a lot more thought and work into it, I'm going to leave you to take a look at the suggestions and commit them. If you have questions about any of them, let me know.

Also, let me know if you have any questions about the suggestion to update the `{% data reusables.support.connect-in-the-forum-bootcamp %}` reusable.

codewithdev committed:
Update content/get-started/quickstart/set-up-git.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev commented on content/get-started/quickstart/set-up-git.md:
```diff
@@ -56,10 +56,10 @@ If you [clone with SSH](/github/getting-started-with-github/about-remote-reposit
-- "[Fork a repository](/articles/fork-a-repo)"
-- "[Be social](/articles/be-social)"
-- {% data reusables.support.connect-in-the-forum-bootcamp %}
+You can create a copy of a repository by forking it and propose the changes that you want to see without affecting the upstream repository. For more inforation see "[Fork a repository](/articles/fork-a-repo)".
```
Yeah, Sometimes I wrote that too quickly 😄 and forgotten about the review on my own.

codewithdev committed:
Update content/get-started/quickstart/set-up-git.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

Benzfccr7 commented on content/get-started/quickstart/fork-a-repo.md:
```diff
@@ -148,7 +148,16 @@ Fork a repository to start contributing to a project. {% data reusables.reposito
 ## Celebrate
 
-You have now forked a repository, practiced cloning your fork, and configured an upstream repository. What do you want to do next?
+You have now forked a repository, practiced cloning your fork, and configured an upstream repository. For more information about cloning the fork and syncing the changes in a forked repository from your computer see "[Set up Git](/articles/set-up-git)".
```
😁

codewithdev committed:
Update content/get-started/quickstart/be-social.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/set-up-git.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev commented on content/get-started/quickstart/create-a-repo.md:
```diff
@@ -79,11 +79,13 @@ Let's commit a change to the *README* file.
-- {% data reusables.support.connect-in-the-forum-bootcamp %}
+Congratulations! You have now created a repository, including a *README* file, and created your first commit on {% data variables.product.product_location %}.
+
+You can now Clone a {% data variables.product.product_name %} repository to create a local copy on your computer. From your local repository you can commit, and create a pull request to update the changes in upstream repository. For more information, see "[Clone a repository](/github/creating-cloning-and-archiving-repositories/cloning-a-repository)" and "[Set up Git](/articles/set-up-git)"
```
Thanks ✨, I will make sure to use the same name when referring to a link within the article.

codewithdev committed:
Update content/get-started/quickstart/create-a-repo.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/create-a-repo.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/fork-a-repo.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/fork-a-repo.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/fork-a-repo.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/be-social.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/be-social.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update content/get-started/quickstart/create-a-repo.md

Co-authored-by: Felicity Chapman <felicitymay@github.com>

codewithdev committed:
Update connect-in-the-forum-bootcamp.md

codewithdev committed:
Update be-social.md

codewithdev committed:
Update create-a-repo.md

codewithdev committed:
Update fork-a-repo.md

codewithdev committed:
Update set-up-git.md

codewithdev commented:
@felicitymay I've updated the internal docs where reusable (connect-in-the-forum-bootcamp.md) is located.

 Since, this reusable is used only on these four articles thus I've applied all the changes that you've mentioned in the comment.
Also I've updated the necessary changes that you've suggested and replaced the community support with just, `{% data reusables.support.connect-in-the-forum-bootcamp %}`

Let me know if there need some certain changes in the support community reusable.

felicitymay commented:
Thanks for making all those changes. 💖 

I'm not sure why the deployment to staging is still showing as pending, I'll update your branch with changes from `main` which should retrigger the deployment. 🤞🏻 

Once we can see the articles on staging, we can have a final check for anything that's slipped through the review, and then merge 😄

felicitymay committed:
Merge branch 'main' into codewithdev-docs-2

codewithdev commented:
> Thanks for making all those changes. 💖

> Once we can see the articles on staging, we can have a final check for anything that's slipped through the review, and then merge 😄

Thats' Great. Waiting for the Final Review. 🕺

codewithdev committed:
Update connect-in-the-forum-bootcamp.md

codewithdev commented:
Waiting for the Final Review 👀

felicitymay commented:
> There needs a little update on the reusable (`docs/data/reusables/support/connect-in-the-forum-bootcamp.md`) and we can remove that extra ". dot"

That's a great point  - well spotted ✨ 

In this case, it would be better to keep the "dot" (period) in the reusable and delete it from the articles. We try to keep reusables as full sentences that can be reused easily. For more information, see [Reusables and variables](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#reusables-and-variables).

I've spotted a couple more tiny things that I missed previously - they're all so small, it's probably quicker for me to add suggestions for them, commit them all, and then your pull request will be ready to merge.

felicitymay commented:
I should have said - apart from the very small things I spotted, this is now looking great. Congratulations on making such good use of the reusable too 👍🏻

felicitymay commented on data/reusables/support/connect-in-the-forum-bootcamp.md:
```diff
@@ -1 +1 @@
-Connect with people around the world in the [{% data variables.product.prodname_gcf %}](https://github.community)
+{% data variables.product.product_name %} has a great support community where you can ask for help and talk to people from around the world. Join the conversation on [Github Support Community](https://github.community/)
```
```suggestion
{% data variables.product.product_name %} has a great support community where you can ask for help and talk to people from around the world. Join the conversation on [Github Support Community](https://github.community/).
```

felicitymay commented on content/get-started/quickstart/set-up-git.md:
```diff
@@ -56,10 +56,10 @@ If you [clone with SSH](/github/getting-started-with-github/about-remote-reposit
+
+Each repository in {% data variables.product.product_name %} is owned by a person or an organization. You can interact with the people, repositories, and organizations by connecting and following them on {% data variables.product.product_name %}. For more information see "[Be social](/articles/be-social)."
+
+{% data reusables.support.connect-in-the-forum-bootcamp %}.
```
```suggestion
{% data reusables.support.connect-in-the-forum-bootcamp %}
```

felicitymay commented on content/get-started/quickstart/fork-a-repo.md:
```diff
@@ -148,10 +148,10 @@ Fork a repository to start contributing to a project. {% data reusables.reposito
+
+Each repository in {% data variables.product.product_name %} is owned by a person or an organization. You can interact with the people, repositories, and organizations by connecting and following them on {% data variables.product.product_name %}. For more information see "[Be social](/articles/be-social)."
+
+{% data reusables.support.connect-in-the-forum-bootcamp %}.
```
```suggestion
{% data reusables.support.connect-in-the-forum-bootcamp %}
```

felicitymay commented on content/get-started/quickstart/create-a-repo.md:
```diff
@@ -79,11 +79,12 @@ Let's commit a change to the *README* file.
+
+Each repository in {% data variables.product.product_name %} is owned by a person or an organization. You can interact with the people, repositories, and organizations by connecting and following them on {% data variables.product.product_name %}. For more information see "[Be social](/articles/be-social)."
+
+{% data reusables.support.connect-in-the-forum-bootcamp %}.
```
```suggestion
{% data reusables.support.connect-in-the-forum-bootcamp %}
```

felicitymay commented on content/get-started/quickstart/be-social.md:
```diff
@@ -70,11 +70,10 @@ Discover interesting projects using {% data variables.explore.explore_github %},
+- To synchronize your {% data variables.product.product_name %} projects with your computer, you can set up Git. For more information see "[Set up Git](/articles/set-up-git)."
+- You can also create a repository, where you can put all your projects and maintain your workflows. For more information see, "[Create a repository](/articles/create-a-repo)."
+- You can fork a repository to make changes you want to see without affecting the original repository. For more information, see "[Fork a repository](/articles/fork-a-repo)."
+- {% data reusables.support.connect-in-the-forum-bootcamp %}.
```
```suggestion
- {% data reusables.support.connect-in-the-forum-bootcamp %}
```

felicitymay reviewed (dismissed):
I'm going to move the "dot" (period) back into the reusable and add a missing "the" but otherwise this looks great and will be ready to merge with these very minor adjustments.

Thanks for your patience with this process ✨

felicitymay committed:
Apply minor updates from code review

codewithdev commented:
Learned many things out from this PR. Thanks, @felicitymay for the prime suggestions and reviews. ✨

felicitymay commented:
> Learned many things out from this PR.

I'm glad it was useful. It's difficult for a reviewer to get the balance right. We don't want to make the process too slow so we often just make changes, but you seemed to want to know more.  ✨
